src/other/landingpage/story.js

```diff
@@ -12,16 +12,52 @@ story.addParameters({
   knobs: { disabled: true },
   changelog: { disabled: true },
   codepreview: { disabled: true },
+  a11y: { disabled: true },
 });
 
+const getFormattedGitCommitMessage = answerJson => {
+  const formattedMessage = answerJson.items[0].commit.message
+    .replace('Publish\n\n', '')
+    .replace(/- /g, '<br/>');
+  return formattedMessage;
+};
+
+const getDateFromGitCommit = answerJson => {
+  const formattedDate = new Date(answerJson.items[0].commit.author.date);
+  return formattedDate.toLocaleString(navigator.language);
+};
+
 story.add('to Pattern Library', () => {
   const wrapper = document.createElement('div');
 
+  const xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    if (this.readyState === 4 && this.status === 200) {
+      const responseJson = JSON.parse(xhttp.responseText);
+      document.querySelector(
+        '#githubResponse'
+      ).innerHTML = getFormattedGitCommitMessage(responseJson);
+      document.querySelector(
+        '#githubResponseDate'
+      ).innerHTML = getDateFromGitCommit(responseJson);
+    }
+  };
+  xhttp.open(
+    'GET',
+    'https://api.github.com/search/commits?q=repo:axa-ch/patterns-library+Publish&sort=author-date&order=desc',
+    true
+  );
+  xhttp.setRequestHeader('Accept', 'application/vnd.github.cloak-preview');
+  xhttp.send();
+
   const template = html`
-    <axa-heading rank="3">Welcome to the Pattern Library!</axa-heading>
+    <axa-heading rank="2">Welcome to the Pattern Library!</axa-heading>
     <axa-text variant="size-3">
       You can find all our Webcomponents here. Check out our Readme below.
     </axa-text>
+    <axa-heading rank="3">Our latest releases:</axa-heading>
```

Comment:
Suggest to rewrite in singular and without colon: '... latest release'

Reply:
But we list multiple releases below. Will change it to "Latest Releases:".

Reply:
The colon feels out of place, since the new topic is already marked by font weight, i.e. we don't just have pure text to express subtitles. And it feels like this should be singular since it has a single date!

But I can approve as-is, and it could be fixed later.